### DISCUSSION TITLE
Alpine 3.13 container images now available for .NET 5.0

### DISCUSSION THREAD

**Discussion by mthalman** [dotnet/dotnet-docker#3684] [Announcements]
# Alpine 3.13 container images now available for .NET 5.0

Alpine 3.13 container images have been released for .NET 5.0. Floating Alpine-based tags now reference Alpine 3.13 images, and previously referenced Alpine 3.12.

.NET Core 2.1 and 3.1 Alpine tags still reference Alpine 3.12. Floating Alpine-based tags for these .NET Core versions will be updated to reference Alpine 3.13 in a later servicing release.

## Details
* https://github.com/dotnet/dotnet-docker/pull/2532

If you make use of a 5.0 floating tag for Alpine, such as `5.0-alpine`, these have been changed to now reference Alpine 3.13 instead of Alpine 3.12.  Be sure to pull this tag on your machine to get the latest image.

The following Docker repos have been updated:

* [dotnet/sdk](https://hub.docker.com/_/microsoft-dotnet-sdk)
* [dotnet/aspnet](https://hub.docker.com/_/microsoft-dotnet-aspnet)
* [dotnet/runtime](https://hub.docker.com/_/microsoft-dotnet-runtime)
* [dotnet/runtime-deps](https://hub.docker.com/_/microsoft-dotnet-runtime-deps)

**Comment by ChaosEngine** [comment]
I see issues with testing projects specifically for alpine 3.13 version. Will provide repro and new issue ASAP...
There - https://github.com/dotnet/dotnet-docker/issues/2607
